src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -661,11 +667,12 @@ void Subgraph::optimizeIR() {
 
     // DataFlow transformations includes AnalyzeBroadcastableInputs pass:
     // we should verify that the received map is aligned with our blocked input shapes
-    OPENVINO_ASSERT((broadcastable_inputs.size() < in_shapes.size()) ||
-                        (!broadcastable_inputs.empty() && broadcastable_inputs.rbegin()->first < in_shapes.size()),
-                    "Incorrect indexes of broadcastable inputs of Subgraph");
+    OPENVINO_ASSERT(
+        (broadcastable_inputs.size() < in_blocked_shapes.size()) ||
+            (!broadcastable_inputs.empty() && broadcastable_inputs.rbegin()->first < in_blocked_shapes.size()),
+        "Incorrect indexes of broadcastable inputs of Subgraph");
     for (const auto broadcastable_input : broadcastable_inputs) {
-        OPENVINO_ASSERT(broadcastable_input.second < in_shapes[broadcastable_input.first].size(),
+        OPENVINO_ASSERT(broadcastable_input.second < in_blocked_shapes[broadcastable_input.first].first.size(),
```

Comment:
Could you please elaborate why we need to this change? Thanks in advance!

Reply:
Thanks for pointing this out. Actually this is a useless change, I reverted it